docs/content/posts/2024-06-14-release-1.19.1.md

```diff
@@ -0,0 +1,171 @@
+---
+title:  "Apache Flink 1.19.1 Release Announcement"
+date: "2024-06-14T00:00:00.000Z"
+aliases:
+- /news/2024/06/14/release-1.19.1.html
+authors:
+- hong:
+  name: "Hong"
+  twitter: "hlteoh2"
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink 1.19 series.
+
+This release includes 44 bug fixes, vulnerability fixes, and minor improvements for Flink 1.19.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12354399).
+
+We highly recommend all users upgrade to Flink 1.19.1.
+
+# Release Artifacts
+
+## Maven Dependencies
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.19.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>1.19.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>1.19.1</version>
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink/tags?page=1&name=1.19.1) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=1.19.1) (ASF repository)
+
+## PyPi
+
+* [apache-flink==1.19.1](https://pypi.org/project/apache-flink/1.19.1/)
+
+# Release Notes
+
+
+        Release Notes - Flink - Version 1.19.1
+
+<h2>        Sub-task
```

Comment:
It's better to remove the subtask part, and move these JIRAs to bug or Improvement part.

Sub-task means it's a subtask JIRA. But for user side, each jira is bugfix or Improvement or new feature.

Reply:
Ok - removed. All subtasks are not relevant to users